csrc/quantization/cutlass_w8a8/scaled_mm_entry.cu

```diff
@@ -123,6 +135,8 @@ bool cutlass_group_gemm_supported(int64_t cuda_device_capability) {
 #if defined CUDA_VERSION
   if (cuda_device_capability == 90) {
     return CUDA_VERSION >= 12030;
+  } else if (cuda_device_capability >= 100) {
```

Comment:
@kushanam while you're in here, could you fix up the comment just above this? It has a misspelling and also says that SM90 is required, which is no longer true with this PR